tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3330 @@
+          pruningIndex: 0
+        metrics:
+          blocksPerSecond: 17
+          referencedBlocksPerSecond: 16.8
+          referencedRate: 98.82352941176471
+        supportedProtocolVersions:
+          - 2
+          - 3
+        protocol:
+          networkName: iota-testnet
+          bech32Hrp: atoi
+          tokenSupply: "2779530283277761"
+          version: 2
+          minPowScore: 1000
+          rentStructure:
+            vByteCost: 500
+            vByteFactorData: 1
+            vByteFactorKey: 10
+        pendingProtocolParameters:
+          - type: 1
+            targetMilestoneIndex: 480
+            protocolVersion: 2
+            params: "0x03040560405060703940304956896"
+          - type: 1
+            targetMilestoneIndex: 485
+            protocolVersion: 2
+            params: "0x03040503045950393045930239392"
+          - type: 1
+            targetMilestoneIndex: 490
+            protocolVersion: 2
+            params: "0x02010290394032948950303939482"
+        baseToken:
+          name: "IOTA"
+          tickerSymbol: "MIOTA"
+          unit: "IOTA"
+          decimals: 0
+          useMetricPrefix: true
+        features:
+          - PoW
+    get-info-response-example-shimmer:
+      value:
+        name: HORNET
+        version: 2.0.0-alpha8
+        status:
+          isHealthy: true
+          latestMilestone:
+            index: 480
+            timestamp: 1617802102
+            milestoneId: "0xb59ff329113b0da14343707450cb28d41fa18b295deabc4beb3fc1b6e70f9d9e"
+          confirmedMilestone:
+            index: 480
+            timestamp: 1617802102
+            milestoneId: "0xb59ff329113b0da14343707450cb28d41fa18b295deabc4beb3fc1b6e70f9d9e"
+          pruningIndex: 0
+        metrics:
+          blocksPerSecond: 17
+          referencedBlocksPerSecond: 16.8
+          referencedRate: 98.82352941176471
+        supportedProtocolVersions:
+          - 2
+          - 3
+        protocol:
+          networkName: shimmer-testnet
+          bech32Hrp: rms
+          tokenSupply: "2779530283277761"
+          version: 2
+          minPowScore: 1000
+          rentStructure:
+            vByteCost: 500
+            vByteFactorData: 1
+            vByteFactorKey: 10
+        pendingProtocolParameters:
+          - type: 1
+            targetMilestoneIndex: 480
+            protocolVersion: 2
+            params: "0x03040560405060703940304956896"
+          - type: 1
+            targetMilestoneIndex: 485
+            protocolVersion: 2
+            params: "0x03040503045950393045930239392"
+          - type: 1
+            targetMilestoneIndex: 490
+            protocolVersion: 2
+            params: "0x02010290394032948950303939482"
+        baseToken:
+          name: "Shimmer"
+          tickerSymbol: "SMR"
+          unit: "SMR"
+          decimals: 6
+          subunit: "glow"
+          useMetricPrefix: false
+        features:
+          - PoW
```

Comment:
```suggestion
          - Pow
```

Reply:
@muXxer can elaborate, we decided to leave it like this for now.